Sources/HTMLKit/Internal/Features/Conversion/Converter.swift

```diff
@@ -645,7 +644,12 @@ extension Converter {
             let indent = String(repeating: "\t", count: (node.level - 1) + (preindent ?? 0))
             
             if let text = text {
-                "\(indent)\"\(text)\"\n"
+                if node.parent?.localName != "pre" {
+                    let cleanText = text.replacingOccurrences(of: "\n", with: "")
+                    "\(indent)\"\(cleanText)\"\n"
+                } else {
+                    "\(indent)\"\"\"\(text)\"\"\"\n"
+                }
```

Comment:
What's this for? And is it a specific case of the string to string conversion or would it be necessary for the decoding in general?